reporter-plugin/src/main/scala/com/virtuslab/indexing/data/IndexingStats.scala

```diff
@@ -0,0 +1,11 @@
+package com.virtuslab.indexing.data
+
+case class IndexingStats(
+    startedAt: Long,
```

Comment:
Let's put the duration and shared index utilization too, even though they can be computed. It will be easier to query. 
Now that I am thinking, we might want  to observe full indexing time (with sannings, as this also takes time and may change), and real indexing time (without pauses, it also shows in the report).

Reply:
Done. I'm not entirely sure that I interpret the data correctly, but at least all the necessary data sources are retrieved.